### PR TITLE
Issue #99: Create event badge skeleton for place card view.

### PR DESCRIPTION
**Pull request by mcomella**
The event appears on one place card and is filled with dummy data. No click handlers yet.

Screenshot:
simulator screen shot nov 8 2016 17 34 50

### CODE REVIEW

File: Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift [hunk 1]
```diff
@@ -144,17 +152,40 @@ class PlaceDetailsCardView: UIView {
...
     private func setupViews() {
-        backgroundColor = Colors.detailsViewCardBackground
+        backgroundColor = Colors.detailsViewCardBackground // cannot be transparent to display shadow
+        contentView.backgroundColor = Colors.detailsViewCardBackground
+
         layer.cornerRadius = 10
+        contentView.layer.cornerRadius = 10
+        contentView.layer.masksToBounds = true
+
+        addSubview(contentView)
```
Comment (fluffyemily) [inline Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift]: Is there any reason why you didn't just insert the new view into the stack view at the very top?

**Comment by mcomella** [comment]
[I want my Prox cards to also include events that will be happening soon because this might make me more interested in visiting the venues.](https://trello.com/c/Zf5kaT3F/195-i-want-my-prox-cards-to-also-include-events-that-will-be-happening-soon-because-this-might-make-me-more-interested-in-visiting-t)

**Comment by mcomella** [comment]
@fluffyemily Please re-review.

I didn't use `eventView.removeArrangedSubview` because, according to the docs, this doesn't actually remove the view from the view hierarchy and it keeps displaying (you need to use `UIView.removeFromSuperview` for that) – hiding the view seemed simpler.

**Comment by fluffyemily** [comment]
Not sure why, but this is what I see when I first run through the app. I am wondering whether some of the unsatisfiable constraints are causing this to break.

screen shot 2016-11-10 at 11 55 42

**Pull request merged**